main.py

```diff
@@ -24,18 +134,33 @@ def mondo_notifier(request):
         Response object using `make_response`
         <https://flask.palletsprojects.com/en/1.1.x/api/#flask.make_response>.
     """
+
+    if not request:
+      logging.error("No request object.")
+      return 'No request object. Processing stopped.'
+
+    if not validate_signature(request):
+      logging.error("Github signature did not validate.")
+      return 'Webhook signature did not validate. Processing stopped.'
```

Comment:
One suggestion is to return an http status code on these error cases. I think it defaults to `200` if none is given. In flask you can return a tuple like `return ("msg", 403)`. I think it just ends up going to the log, but might be useful for filtering log messages. I think it would also flag it on the github side in the webhook history page.

Reply:
done. updated PR, please re-review